server/cmwell-irw/src/test/scala/cmwell/irw/IRWCassSpec.scala

```diff
@@ -257,22 +260,25 @@ trait IRWCassSpec extends AsyncFlatSpec with Matchers with IRWServiceTest {
   }
 
   "write fat infoton (with more than 65K fields/values)" should "succeed" in {
-    val lotsOfFields = Seq.tabulate(0xFFFF * 2){ n =>
+    val lotsOfFields = Seq.tabulate(0xFFFF * 2) { n =>
       s"field$n" -> Set[FieldValue](FString(s"value$n"))
     }.toMap
 
     val fatFoton = ObjectInfoton("/irw/xyz/fatfoton1", "dc_test", None, lotsOfFields, None)
-
-    irw.writeAsync(fatFoton).flatMap{ _ =>
-      scheduleFuture(10.seconds) {
-        irw.readPathAsync("/irw/xyz/fatfoton1").map{
-          case FullBox(readInfoton) => readInfoton shouldBe fatFoton
-          case EmptyBox => fail("/irw/xyz/fatfoton1 was not found")
-          case BoxedFailure(e) => fail("error occured",e)
-        }
+    irw.writeAsync(fatFoton)
+    cmwell.util.concurrent.spinCheck(100.millis, true)(irw.readPathAsync("/irw/xyz/fatfoton1")) {
+      res => res match {
+        case FullBox(readInfoton) => readInfoton == fatFoton
+        case _ => false
       }
+    }.map ( res  =>
+      withClue(res) (res match {
```

Comment:
as a convention, use {} for functions that spawn more than one line.

Reply:
I have several cases, so I must have {},,right?